cmd/installer/cli/install.go

```diff
@@ -153,6 +161,85 @@ func InstallCmd(ctx context.Context, name string) *cobra.Command {
 	return cmd
 }
 
+func installCmdExample() string {
+	if os.Getenv("ENABLE_V3") != "1" {
+		return ""
+	}
+
+	return `
+  # Install on a Linux host
+  replicated install \
+      --target linux \
+      --data-dir /opt/embedded-cluster \
+      --cidr 10.244.0.0/16 \
+      --license ./license.yaml \
+      --yes
+
+  # Install into an existing Kubernetes cluster
+  replicated install \
+      --target kubernetes \
+      --k8s-admin-console-ns replicated-admin \
```

Comment:
this flag isn't in the help menu.

we're not yet prefixing with k8s, so i think this could be `--admin-console-namespace` and be available for kubernetes only.

Reply:
changed to --kubeconfig